static/app/views/performance/newTraceDetails/traceHeader/index.tsx

```diff
@@ -0,0 +1,210 @@
+import {useCallback} from 'react';
+import styled from '@emotion/styled';
+
+import Breadcrumbs from 'sentry/components/breadcrumbs';
+import ButtonBar from 'sentry/components/buttonBar';
+import DiscoverButton from 'sentry/components/discoverButton';
+import {HighlightsIconSummary} from 'sentry/components/events/highlights/highlightsIconSummary';
+import FeedbackWidgetButton from 'sentry/components/feedback/widget/feedbackWidgetButton';
+import * as Layout from 'sentry/components/layouts/thirds';
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {EventTransaction} from 'sentry/types/event';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import type EventView from 'sentry/utils/discover/eventView';
+import {SavedQueryDatasets} from 'sentry/utils/discover/types';
+import type {UseApiQueryResult} from 'sentry/utils/queryClient';
+import type RequestError from 'sentry/utils/requestError/requestError';
+import {useLocation} from 'sentry/utils/useLocation';
+import {hasDatasetSelector} from 'sentry/views/dashboards/utils';
+import {ProjectsRenderer} from 'sentry/views/explore/tables/tracesTable/fieldRenderers';
+
+import type {TraceMetaQueryResults} from '../traceApi/useTraceMeta';
+import TraceConfigurations from '../traceConfigurations';
+import type {TraceTree} from '../traceModels/traceTree';
+import {useHasTraceNewUi} from '../useHasTraceNewUi';
+
+import {getTraceViewBreadcrumbs} from './breadcrumbs';
+import {Meta} from './meta';
+import {Title} from './title';
+
+interface TraceMetadataHeaderProps {
+  metaResults: TraceMetaQueryResults;
```

Comment:
I renamed some of this in my PR, feel free to keep it the same here, but I find the `Results` naming weird and figured we should just remove it and call this meta

Reply:
Yeah, I'll rename them, I don't have strong opinions on the naming of this type